test/unit/notify/send-email.test.js

```diff
@@ -102,16 +104,27 @@ describe('send email', () => {
     expect(notifyClientMockInstance.sendEmail).toHaveBeenCalledTimes(1)
   })
 
-  test('should call notifyClient.sendEmail with correct parameters when a valid message is received', async () => {
+  test('should not call notifyClient.sendEmail with empty reference', async () => {
     await sendEmail(mockContext, mockMessage)
 
     const notifyClientMockInstance = notifyClient.mock.instances[0]
-    expect(notifyClientMockInstance.sendEmail).toHaveBeenCalledWith(env.notifyEmailTemplateId, env.notifyEmailAddress, {
+    expect(notifyClientMockInstance.sendEmail).not.toHaveBeenCalledWith(env.notifyEmailTemplateId, env.notifyEmailAddress, {
       personalisation: flatten(mockMessage),
       reference: defaultReference
```

Comment:
This is not misleading -- I had originally setup `defaultReference` to represent what the default behaviour of this was, i.e. previously it was  `''` but now it is `mockReference`

I would change `defaultReference` to `mockReference` and in the test where you're testing that `reference` isn't `''`, just explicitly have an empty string as the compared `reference` value